src/root/POSRootChainManager.ts

```diff
@@ -90,6 +103,32 @@ export default class POSRootChainManager extends ContractsBase {
     return this.web3Client.send(txObject, _options)
   }
 
+  async approveMaxERC20(rootToken: address, options?: SendOptions) {
+    if (!this.erc20Predicate) {
+      throw new Error('posERC20Predicate address not found. Set it while constructing MaticPOSClient.')
+    }
+    const txObject = this.getPOSERC20TokenContract(rootToken, true).methods.approve(
+      this.erc20Predicate,
+      '0xffffffffffffffffffffffffffffffff'
```

Comment:
max allowance should be 32 bytes so `0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff`

Reply:
done